src/main/java/org/apache/maven/plugins/javadoc/JavadocUtil.java

```diff
@@ -781,9 +781,13 @@ protected static void invokeMaven(
         request.setBaseDirectory(projectFile.getParentFile());
         request.setPomFile(projectFile);
         request.setGlobalSettingsFile(globalSettingsFile);
-        request.setUserSettingsFile(userSettingsFile);
+        if (userSettingsFile.isFile()) {
```

Comment:
more needed: UT will NPE as it tests with null value while in maven-core/embedder this value is never null.